002-issue-workflow.md

```diff
@@ -2,7 +2,7 @@
 
 The process of handing incoming issues is the following:
 
-![Issue workflow schema](images/002-issue-workflow.svg)
+![Issue workflow schema](docs/images/002-issue-workflow.svg)
```

Comment:
I don't think we should have `docs` directory in `docs` repository.

Reply:
See [Documentation is one of the most important parts of Yii. - 
Package documentation](https://github.com/yiisoft/docs/blob/master/014-docs.md#package-documentation)

Reply:
That applies to packages and this repository is not a package. It is definitive guide.